src/client/styles/components/Drbb/Drbb.scss

```diff
@@ -79,27 +80,42 @@ a.drbb-read-online {
   background-color: #fff;
   border: 0.08333rem solid #d0343a;
   border-radius: 0.13333rem;
+  box-sizing: border-box;
   color: #d0343a;
-  cursor: pointer;
-  display: inline-block;
   font-size: 0.8rem;
   font-weight: normal;
+  height: 33px;
   letter-spacing: 0.06rem;
-  text-decoration: none;
+  margin-right: 4%;
   padding: 2% 3%;
+  text-decoration: none;
 }
 
 a.drbb-download-pdf {
-  float: right;
   color: #000000;
   fill: currentColor;
   padding: 2% 0;
+  font-size: 0.8rem;
+  font-weight: normal;
+  letter-spacing: 0.06rem;
 
   svg {
-    padding-left: 5px;
+    padding-right: 5px;
```

Comment:
Is the padding for the space between the download link and the icon?  If so I'd put the svg in a div and put the padding on that, rather than adding it to the svg on its own.  But if the margin-bottom is for that mysterious floating thing, we haven't figured out how to fix it the "right" way yet either so ... also happy to have whatever works.